thriftbp/client_pool.go

```diff
@@ -611,13 +601,19 @@ func (p *clientPool) pooledCall(ctx context.Context, method string, args, result
 	return client.Call(ctx, method, args, result)
 }
 
-func (p *clientPool) getClient() (Client, error) {
+func (p *clientPool) getClient() (_ Client, err error) {
+	defer func() {
+		clientPoolGetsCounter.With(prometheus.Labels{
+			"thrift_pool":    p.slug,
+			"thrift_success": strconv.FormatBool(err == nil),
```

Comment:
note: the spec only defined `thrift_pool` label, but I think `thrift_success` label here would be really useful.

Reply:
What does "success" mean here though, succesfully getting a client from the pool?

Reply:
Yes. do you have a better name to suggest?